src/CppUTestExt/MockActualCall.cpp

```diff
@@ -643,6 +639,7 @@ unsigned int MockActualCallTrace::returnUnsignedIntValueOrDefault(unsigned int)
 
 MockActualCall& MockActualCallTrace::onObject(void* objectPtr)
 {
+    traceBuffer_ += " onObject:";
```

Comment:
The address of the object used to be stuck onto the CallOrder like so:
`withCallOrder: 10x4536123`

One might use newlines for every parameter (with an indent) or stick on everything seperated by spaces like now. Also, a space might be after the colon, but if so, I feel it should be done consistently.